### PR TITLE
Handle unrecognised account errors

### LINKED ISSUE #347: Desktop wallet locks and refuses to unlock. No clear way to reset the wallet.
The issue tracker is only for reporting bugs or asking for feature requests.
If you need technical assistance for using Trinity Wallet, please consult the Trinity [documentation](https://docs.iota.works/trinity) and [knowledge base](https://iotaledger.atlassian.net/wiki/spaces/trinity/overview). For further assistance, please visit the #help channel on [Discord](https://discord.gg/7Gu2mG5).
If you have general questions on IOTA you can go to https://iota.stackexchange.com/, https://forum.helloiota.com/, or browse Discord channels (https://discord.gg/C88Wexg).

### Bug description
A general description of the bug.

### Environment Specs
What device are you running Trinity on?

- Manufacturer: 
- Model: 
- OS version: Linux

What build of Trinity are you running? Please provide the full version (you can find it in Settings > About Trinity).

- Trinity version: Desktop 0.3.3

### Steps To Reproduce
1. imported wallet seed
2. set password
3. allow the application to lock such that it requests a password.

### Expected behaviour
unlock application with password

### Actual behaviour
password no longer accepted after locking. 

### Errors
"This password was not recognised. Please try again."

### Description
The ability to easily reset the desktop wallet.

### Motivation
Deleting the appImage doesn't reset the wallet, and I am still locked out. 

### Requirements
Create a list of what you want this feature request to fulfill.

### Open Questions (optional)
Anything you want to discuss.

### Am I planning to do it myself with a PR?
No.

### PR DESCRIPTION
**Pull request by delki8**
… wallet. #347 

# Description

- I found a problem that might be this one, the account name wasn't being recognized for my seed. (# How Has This Been Tested?)

Fixes #347 

## Type of change

- Bug fix (a non-breaking change which fixes an issue)

# How Has This Been Tested?

- That happened because in my daily use trinity client I gave one name to my account, let's say 'Account #1', and in the development client, for the same seed, I gave the name 'Account #2'. Then my password was correct but the seed wasn't being recoverd because the account name associated with the dev client was incorrect, the app was failing silently and what I'm doing here is to show an error to the user.

# Checklist:

- [ x ] My code follows the style guidelines for this project
- [ x ] I have performed a self-review of my own code
- [ x ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/shared/locales/pt-BR/translation.json
Comment (rajivshah3) [inline src/shared/locales/pt-BR/translation.json]: Please don't make manual changes to the translation files (except `locales/en/translation.json`). These are automatically generated and will likely be overwritten when translations are synced from [Crowdin](https://crowdin.com/project/iota-trinity-wallet). Instead, you can suggest changes there and they will automatically be synced back to this repository.

File: src/shared/locales/en/translation.json
Comment (rajivshah3) [inline src/shared/locales/en/translation.json]: IOTA aims to use UK English as much as possible, so `unrecognized` should be `unrecognised` and `recognized` should be `recognised`.

**Comment by delki8** [comment]
It seems that I mess up... I was trying to address the #347 issue that is already opened... Sorry for this.

**Review by rajivshah3 — CHANGES_REQUESTED** [review]
Hi, thanks for the PR! Sorry that the documentation wasn't clear on what needs to be done for localization. Please see the comments below

**Comment by rajivshah3** [comment]
> It seems that I mess up... I was trying to address the #347 issue that is already opened... Sorry for this.

@delki8 this seems fine except for the localization parts

**Comment by delki8** [comment]
@rajivshah3 Ok, So I changed the name of the key to match the UK English (I didn't know that there were a difference) and changed the en/translation.json also with the UK spelling. I'm also reverting the change made directly at ptBr translation file and didn't bother to fix the other files to the UK spelling because they will be replaced either way.

And let me know if there is anything left :)

File: src/shared/locales/en/translation.json
Comment (rajivshah3) [inline src/shared/locales/en/translation.json]: Sorry, I forgot one thing in the earlier review. Could you change the last part to `Consider resetting your wallet.` or `Please reset the wallet.`?

**Review by rajivshah3 — CHANGES_REQUESTED** [review]

File: src/shared/locales/en/translation.json
Comment (delki8) [reply src/shared/locales/en/translation.json]: Not a problem.

Comment (rajivshah3) [reply src/shared/locales/en/translation.json]: Just realized that the Advanced Settings can't be accessed if you can't log in 😅 `Please reset the wallet` might be better here in that case

Comment (delki8) [reply src/shared/locales/en/translation.json]: hehe, I should have thought of that as well

File: src/shared/locales/en/translation.json
Comment (rajivshah3) [inline src/shared/locales/en/translation.json]: I just realized Advanced Settings isn't accessible if you can't log in. Could you change the last part to `Please reset the wallet.`?

**Review by rajivshah3 — CHANGES_REQUESTED** [review]
Once the string is fixed and conflicts are resolved, this should be ready to merge (pending review from @rihardsgravis)

File: src/shared/locales/en/translation.json
Comment (delki8) [reply src/shared/locales/en/translation.json]: Sure, I'll do it

**Review by rajivshah3 — APPROVED** [review]

**Comment by rajivshah3** [comment]
Thanks @delki8 ! I've asked Rihards if he can review

**Comment by laumair** [comment]
@delki8 LGTM, thanks!   👍

**Pull request merged**
